docs/data/date-pickers/fields/fields.md

```diff
@@ -33,23 +23,17 @@ The field components requires an adapter imported from `@mui/x-date-pickers` or
 
 All fields to edit a range are available in a single input version and in a multi input version.
 
-:::info
-The single input version of the Time Range Field and the Date Time Range Field will soon be available.
-:::
-
 {{"demo": "DateRangeFieldExamples.js", "defaultCodeOpen": false}}
 
 ### Usage with the pickers
 
-The picker components (`DatePicker`, `DestkopDatePicker`, `MobileDatePicker`, `DateTimePicker`, ...) do not use the field components yet.
-
-New pickers using them will be released in a few months.
-
-## Form props
+The picker components (e.g. `DatePicker`) are using the field to handle their input editing.
 
-The standard form attributes (`disabled`, `readOnly`) are supported by all the field components:
+:::warning
+The single input range fields are not supported on the range pickers yet (you can't use `SingleInputDateRangeField` in `DateRangePicker`). It's coming.
 
-{{"demo": "FormProps.js", "defaultCodeOpen": false}}
```

Comment:
When I wrote this page, we had no detailed page for each field.
Now each field has an equivalent of this section, so I don't think it's useful to keep it here.